todoapp/todos/views.py

```
@@ -26,28 +26,22 @@ class TodoAPIViewSet(rest_viewsets.ModelViewSet):
     """
 
-    permission_classes = [rest_permissions.AllowAny]
+    authentication_classes = [rest_authentication.TokenAuthentication]
+    permission_classes = [rest_permissions.IsAuthenticated, users_permissions.IsOwner]
 
     def get_serializer_class(self):
         if self.action == "create":
             return todos_serializers.TodoCreateSerializer
         return todos_serializers.TodoDetailSerializer
 
     def get_user(self):
```

Ruff (0.12.2)

Missing return type annotation for public function `get_user`

(ANN201)